Reference/Searching/Examine/examine-manager.md

```diff
@@ -52,5 +52,4 @@ The indexing methds available are:
 	void ReIndexNode(XElement node, string type);
 	void ReIndexNode(XElement node, string type, IEnumerable<BaseIndexProvider> providers);
 
-**Indexing documentation coming soon!**
-
+**Indexing documentation coming soon!**
```

Comment:
We try to avoid "coming soon" and "this is missing" in the documentation.

Reply:
Agreed. Removed it for now as it is covered in other parts but could probably do with it's own section at some point.

Reply:
👍